src/bci_build/package.py

```diff
@@ -1052,10 +1055,18 @@ def uid(self) -> str:
     @property
     def build_tags(self) -> List[str]:
         tags = []
+        # deconflict tags by the stability ordering
+        _stability_tag_ordering = (None, "stable", "oldstable")
+        relsuffix = "%RELEASE%"
+        if self.stability_tag and self.stability_tag in _stability_tag_ordering:
+            relsuffix = f"{_stability_tag_ordering.index(self.stability_tag)}.%RELEASE%"
         for name in [self.name] + self.additional_names:
-            for ver_label in [self.version_label] + self.additional_versions:
+            ver_labels = [self.version_label]
+            if self.stability_tag:
+                ver_labels = [self.stability_tag] + ver_labels
+            for ver_label in ver_labels + self.additional_versions:
                 tags += [f"{self._registry_prefix}/{name}:{ver_label}"] + [
-                    f"{self._registry_prefix}/{name}:{ver_label}-%RELEASE%"
+                    f"{self._registry_prefix}/{name}:{ver_label}-{relsuffix}"
```

Comment:
We don't want that necessarily, because now we have tags like this:
```
#!BuildTag: opensuse/bci/golang:stable
#!BuildTag: opensuse/bci/golang:stable-1.%RELEASE%
#!BuildTag: opensuse/bci/golang:1.20
#!BuildTag: opensuse/bci/golang:1.20-1.%RELEASE%
```
The last entry is not correct though (well, technically it is not wrong, but imho it's missleading as it pretends to be a different build than it actually is)

Reply:
The last entry is correct ( we need to deconflict the release numbers). just setting it on stable is not enough

Reply:
Ah, you're right. That makes sense.